### DISCUSSION TITLE
Sensor error

### DISCUSSION THREAD

**Discussion by joshg1994** [NightscoutFoundation/xDrip#3435] [Q&A]
There has been a 20 min sensor error. Using Samsung ultra 23 and dexcom g7
![Screenshot_20240421_113912_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/157648698/0bc2bc77-748b-4135-b25b-778a80731643)

**Comment by joshg1994** [comment]
Wondering what troubleshooting options are

**Comment by Navid200** [comment]
Thanks for contacting us.  I have converted this to a discussion.  Please use discussions to inform us of any concerns you may have.  If needed, it will be converted to an issue.

A sensor error is usually related to the sensor rather than the connection between G7 and xDrip.
It could be caused by a real damage to the sensor or it could also be caused by a severe compression low.

Is it still ongoing?
If you ever wonder if there could be something wrong with xDrip, you can always switch to the Dexcom app.
All you need to do for that is to disable collection in xDrip first: https://navid200.github.io/xDrip/docs/Stop-xDrip.html
Then, you can set up Dexcom app to collect from your G7.
If you do that and Dexcom app works fine consistently and xDrip fails consistently, please let us know.

But, if you see sensor errors using Dexcom app as well, there is not much we can change in xDrip to address that.

**Reply by joshg1994** [reply]
Thank you, I ended up ending the sensor early, unfortunately, and when I replaced it with a new one. It began to work perfectly fine. Therefore I presume it was a sensor issue, rather xdrip
